src/main/resources/static/assets/js/sdh.js

```diff
@@ -0,0 +1,68 @@
+(function() {
+	var app = new Vue({
+		el: '#contents',
+		data: {
+			fields: [{key:'domain', sortable:true}, {key:'date_last_assessed', sortable:true}, {key:'clinician_priority', sortable:true}, {key:'patient_readiness', sortable:true}],
+			filters: {
+			      domain: '',
+			      date_last_assessed: '',
+			      clinician_priority: '',
+			      patient_readiness: ''
+			    },
+	        items: [
+	            { domain: 'Food Security', date_last_assessed: '01/01/2018', clinician_priority: 'High', patient_readiness: 'Not Ready' },
+	            { domain: 'Transportation', date_last_assessed: '01/01/2019', clinician_priority: 'Low', patient_readiness: 'Motivated'},
+	            { domain: 'Housing Instability', date_last_assessed: '10/11/2018', clinician_priority: 'Medium', patient_readiness: 'Ready' },
+	            { domain: 'Utilities', date_last_assessed: '02/01/2019', clinician_priority: 'High', patient_readiness: 'Ready' },
+	            { domain: 'Interpersonal Violence', date_last_assessed: '12/01/2018', clinician_priority: 'Medium', patient_readiness: 'Motivated' },
+	            { domain: 'Education', date_last_assessed: '04/04/2018', clinician_priority: 'Medium', patient_readiness: 'Not Ready' },
+	            { domain: 'Financial Strain', date_last_assessed: '03/17/2012', clinician_priority: 'Low', patient_readiness: 'Ready' }
+	          ]
+		},
+		methods: {
+			_priorityVariant (priority) {
+				if (priority == 'High') {
+					return 'danger';
+				} else if (priority == 'Medium') {
+					return 'warning';
+				} else if (priority == 'Low') {
+					return 'success';
+				}
+		    },			
+			_readinessVariant (readiness) {
+				if (readiness == 'Not Ready') {
+					return 'danger';
+				} else if (readiness == 'Motivated') {
+					return 'warning';
+				} else if (readiness == 'Ready') {
+					return 'success';
+				}
+		    }			
+		},
+		computed: {
+		    filtered () {
+		        const filtered = this.items.filter(item => {
+		          return Object.keys(this.filters).every(key =>
+		              String(item[key].toLowerCase()).includes(this.filters[key].toLowerCase()));
```

Comment:
Can you explain the use of the string constructor here? If `item[key]` has a `toLowerCase()` method, isn't it already a string?

Reply:
Yeah, it's probably not needed. The filtered method was copy/pasted from a Stackoverflow example. I added the toLowerCase() part when I realized the filter was case sensitive. I'll see if I can remove it. One of the columns will be a date when it comes from the database, but will likely be converted to a string for display anyway.